docs/tutorials/quick-start/index.rst

```diff
@@ -702,25 +756,106 @@ certificate.
         Name:       letsencrypt-prod
       Secret Name:  quickstart-example-tls
     Status:
-      Acme:
-        Order:
-          URL:  https://acme-v02.api.letsencrypt.org/acme/order/45980184/182533829
       Conditions:
-        Last Transition Time:  2018-11-19T19:16:10Z
-        Message:               Certificate issued successfully
-        Reason:                CertIssued
-        Status:                True
-        Type:                  Ready
-        Last Transition Time:  <nil>
-        Message:               Order validated
-        Reason:                OrderValidated
+        Last Transition Time:  2019-01-09T13:52:05Z
+        Message:               Certificate does not exist
+        Reason:                NotFound
         Status:                False
-        Type:                  ValidateFailed
+        Type:                  Ready
+    Events:
+      Type    Reason        Age   From          Message
+      ----    ------        ----  ----          -------
+      Normal  Generated     18s   cert-manager  Generated new private key
+      Normal  OrderCreated  18s   cert-manager  Created Order resource "quickstart-example-tls-889745041"
+
+You can monitor the progress of the ACME Order by running ``kubectl describe``
```

Comment:
rather than "monitor the progress", slightly better wording here would be "see the current state of the order" - as `kubectl describe` is really only providing a snapshot in time, not a live/update of the progress, and "monitor the progress" sort of infers you'll know when it's done from this command, which I don't think is the case.